plugins/config/config.go

```diff
@@ -2,29 +2,156 @@ package config
 
 import (
 	"fmt"
-	"strings"
 
 	"github.com/dokku/dokku/plugins/common"
+	configenv "github.com/dokku/dokku/plugins/config/src/configenv"
```

Comment:
no need to alias

Reply:
also thinking maybe this should go at the root of the package. not sure why it's a subpackage

Reply:
I was trying to isolate it from the dokku-specific stuff but I can move it to the root

Reply:
Perhaps this could be moved to an independent repo and included as a vendored dependency. @josegonzalez thoughts?

Reply:
Whats this dependency do exactly?

Reply:
The idea was that it was a config type that wasn't linked to dokku internals while the config package had the flat helpers which mimic what was in the old `functions` file.

I'm all for eliminating the subpackage, it has a dumb name and I was on the fence about it anyhow. It was more relevant when it handled dotenv parsing/serialization itself but I got the changes into `godotenv` we needed so it doesn't do that anymore.